food-truck-frontend/public/global-styles.css

```diff
@@ -8,6 +8,10 @@ body {
     text-align: justify
 }
 
+.hidden {
+    display: none !important;
```

Comment:
I've never seen `!important` before

Reply:
I have a vague recollection of it from my various plunges into the HTML abyss. I think it means 'hey, inline CSS, you're not allowed to override this'.

Reply:
Yeah, if we have something of class hidden, we want it to never be visible, even if a more specific CSS selector would normally override it